pkg/vat/vat.go

```diff
@@ -38,10 +38,10 @@ type Stream interface {
 }
 
 type ClientProvider interface {
-	// Client returns the client capability to be exported.  It is called
+	// CapClient returns the client capability to be exported.  It is called
 	// once for each incoming Stream, so implementations may either share
 	// a single global object, or instantiate a new object for each call.
-	Client() *capnp.Client
+	CapClient() *capnp.Client
```

Comment:
Any particular reason for the name change?  If we go with this, we'll have to update the methods in `pkg/client/client_test.go` and `pkg/cap/pubsub/server.go`.